data-raw/babies.R

```diff
@@ -0,0 +1,28 @@
+## code to prepare `babies` dataset goes here
```

Comment:
This file should be deleted and the contents moved to `data-raw/process_data_sets.R`